src/main/java/io/kamax/matrix/client/AMatrixHttpClient.java

```diff
@@ -258,6 +272,16 @@ protected URI getClientPath(String action) {
         }
     }
 
+    protected URI getMediaPathWithAccessToken(String action) {
```

Comment:
- Has nothing to do with the login feature
- No media retrieval requires a token if following the CS spec. While I would definitely add this down the road, it's out of scope here and premature I think